LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 Turaga Lab
+Copyright (c) 2020 Diptodip Deb
```

Comment:
Did I do this? Should we change this to Chromatix Team or Turaga Lab or something again?